src/pages/index.tsx

```diff
@@ -71,35 +26,138 @@ function Home() {
       title={`Hello from ${siteConfig.title}`}
       description="Description will go into a meta tag in <head />"
     >
-      <header className={clsx('hero hero--primary', styles.heroBanner)}>
-        <div className="container">
-          <h1 className="hero__title">{siteConfig.title}</h1>
-          <p className="hero__subtitle">{siteConfig.tagline}</p>
-          <div className={styles.buttons}>
-            <Link
-              className={clsx(
-                'button button--outline button--secondary button--lg',
-                styles.getStarted,
-              )}
-              to={useBaseUrl('docs/')}
-            >
-              Get Started
-            </Link>
+      <main className={styles.mainPageWrapper}>
+        <section className="container">
+          <Figure>
+            <img className="lg:order-1 bg-contain" src={useBaseUrl('img/time-savings.png')} alt="time savings" />
+            <figcaption className="flex flex-col justify-center lg:order-2">
+              <h3>Time Savings</h3>
```

Comment:
Do not skip heading ranks (see the other comment on that issue next to the `<h4 className="mb-2">Run Drill4J</h4>`)